### PR TITLE
add Streamx RedisSink doc file

### PR DESCRIPTION
**Pull request by monrg**
add Streamx RedisSink doc file

### CODE REVIEW

File: docs/connector/redis.md [hunk 1]
```diff
@@ -1,6 +1,364 @@
...
+
+```yaml
+#redis sink 配置
+redis.sink:
+  host: 127.0.0.1 #必须参数
+  #选填参数
+  port: 6379
+  database: 2
+  password: 
+  connectType: jedispool #可选参数：jedispool（默认）|sentinel|cluster
```
Comment (wolfboys) [inline docs/connector/redis.md:187]: jedispool need rename to jedisPool

**Review by wolfboys — COMMENTED** [review]
jedispool need rename to jedisPool

File: docs/connector/redis.md (hunk 1 above)
Comment (monrg) [reply docs/connector/redis.md]: finish

**Comment by wolfboys** [comment]
LGTM.

**Pull request merged**